ui/.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Babel is a JavaScript compiler. Basically allows you to use modern syntax (ES6) and then compiles that down to something usable in the browser.